### DISCUSSION TITLE
Error when reading from an input file Selector.in Sink Informations !

### DISCUSSION THREAD

**Discussion by Dumc23** [phydrus/phydrus#68] [Q&A]
Soil moisture content at different depths of the profile is defined, but the final result shows a running error.
one                                                                    
 getdat and gettim called but not available
 getdat and gettim called but not available
 getdat and gettim called but not available
 getdat and gettim called but not available
 getdat and gettim called but not available
 getdat and gettim called but not available
 getdat and gettim called but not available
 reading nodal information
 reading material information
 generating materials
 reading time information
 reading sink information
 Error when reading from an input file Selector.in Sink Informations !

**Comment by martinvonk** [comment]
Can you post the (model) files here as well?
